tests/test_multi_conductor.py

```diff
@@ -0,0 +1,67 @@
+import pint
+from numpy import array
+from numpy.testing import assert_array_almost_equal
+
+from carsons import MultiConductorCarsonsEquations, calculate_impedance
+from tests.helpers import MultiLineModel
+from tests.test_overhead_line import OHM_PER_MILE_TO_OHM_PER_METER
+
+ureg = pint.UnitRegistry()
+
+feet = ureg.feet
+inches = ureg.inches
+miles = ureg.miles
+ohms = ureg.ohms
+kft = ureg.feet * 1000
+
+
+def test_triplex_phased_cable():
+    """
+    Test against 3/0 triplex NS75 aluminum conductor cable.
+    """
+    phases = 'ABC'
+    phase_conductor = {
+        'resistance': (0.611 * (ohms / miles)).to('ohm / meters').magnitude,
+        'gmr': (0.014 * feet).to('meters').magnitude,
+        'wire_positions': (0, 5),
+        'radius': (0.464 / 2 * inches).to('meters').magnitude,
+        'insulation_thickness': 0.00137,
+    }
+    multi_line_model = MultiLineModel({ph: phase_conductor for ph in phases})
+    carsons_model = MultiConductorCarsonsEquations(multi_line_model)
+
+    assert_array_almost_equal(calculate_impedance(carsons_model), array(
+        [[4.389e-4 + 1j*9.200e-4, 5.922e-5 + 1j*8.277e-4,
+          5.922e-5 + 1j*8.277e-4],
+         [5.922e-5 + 1j*8.277e-4, 4.389e-4 + 1j*9.200e-4,
+          5.922e-5 + 1j*8.277e-4],
+         [5.922e-5 + 1j*8.277e-4, 5.922e-5 + 1j*8.277e-4,
+          4.389e-4 + 1j*9.200e-4]]
+    ), decimal=4)
+
+
+def test_triplex_secondary():
+    """
+    Test against 1/0 AA triplex secondary cable from example 11.3
+    in Kersting's book.
```

Comment:
should be specific about which book / edition, so people can look it up easily

Reply:
yeah ok. I also wanted to have tests for duplex and quadruplex cables, but there isn't any examples with those cables...